ChangeLog

```diff
@@ -1,3 +1,53 @@
+3.19.0:
+	- '-N/--negate now prevents persistent classes from being defined'
+	  (ENT-5886)
+	- 'null' JSON value is now handled as empty data in augments/host-specific data
+	  (ENT-7434)
+	- Add Wants cf-postgres.service to cf-apache.service in systemd configuration
+	  (ENT-5125)
+	- Added a new common control attribute 'system_log_level'
+	  For specifying the minimum log level required for log messages to
+	  go to the system log. (ENT-7594)
+	- Added a test leveraging variablesmatching() to check that the agent
+	  doesn't crash when variables defined from a module do not have
+	  explicit tags (ENT-7724)
+	- Added example illustrating data_sysctlvalues() behavior (CFE-3783)
+	- Added support for cfbs managed policy set (ENT-7709)
+	- Allow trailing comma in policy argument list (CFE-3734)
+	- Changed cf-key option --print-digest to take an optional argument.
+	  cf-key now defaults to the public key file in workdir, if no argument
+	  is specified or an empty string is given as the argument. (CFE-3682)
+	- Cached functions are now always called inside promises with 'iflapsed => "0"'
+	  (CFE-3754)
+	- Custom promises can now use the 'handle' attribute (CFE-3439)
+	- Don't fail on new file creation when backups are enabled
+	  (CFE-3640)
+	- Enabled `depends_on` for custom promise types (CFE-3438)
+	- Enabled `with` attribute for custom promise types (CFE-3441)
```

Comment:
I have a vague memory that these cannot be back-ticks but `'` in stead :thinking: